Sources/StreamChat/Config/StreamRuntimeCheck.swift

```diff
@@ -38,4 +38,9 @@ public enum StreamRuntimeCheck {
     ///
     ///  Enables using our legacy web socket connection.
     public static var _useLegacyWebSocketConnection = false
+    
+    /// For *internal use* only
+    ///
+    /// Enables reusing unchanged converted items in database observers.
+    public static var _isDatabaseObserverItemReusingEnabled = true
```

Comment:
Additionally, we probably could set it to false by default. If we fingd any issue with this, we will need to tell every customer to disable this 🤔